.gitignore

```diff
@@ -12,3 +12,7 @@ dist
 *.profdata
 **/coverage
 .cache
+
+# ignore python data
+__pycache__
+.pytest_cache
```

Comment:
Pytest_cache can be removed now from .gitignore because we do not use pytest anymore.